### DISCUSSION TITLE
Integration of PHP frontend app (with Angular v1.4) into React.js main SPA project as a microfrontend.

### DISCUSSION THREAD

**Discussion by sadek-hussein** [webpack/webpack#18619] [General]
I have a front-end project created using PHP, and inside it, we are using Angular v1.4 for building and routing. My point is, we are now working to add a new technology stack like React.js. So, I have created a React.js main SPA project and added the authentication part for this main SPA. All existing applications created with React are now added to the main SPA as microfrontends. However, I still have one pending part: this PHP project. How do I implement it as a microfrontend and use it in the main SPA project?

**Comment by alexander-akait** [comment]
Sorry, please use Stack Overflow for such questions, thank you
